test-requirements.txt

```diff
@@ -1,4 +1,5 @@
 cryptography
 httpretty>=0.9.6
 parameterized>=0.7.0
-coverage>=5.0.3
```

Comment:
One more thing, maybe we should be explicitly installing coverage rather relying transitive dep from pytesy (my guess)

Reply:
pytest-cov still requires coverage<5, and I daresay will always use coverage, so I'd rather pytest-cov pulling it in so the correct version is installed.

Reply:
Due to pip's resolver, if we include coverage and get out of step, we could hit the issue of installing a version of coverage that pytest-cov does not support.